backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/spreadsheet/process/ExportToSpreadsheetProcess.java

```diff
@@ -42,6 +43,26 @@
 public class ExportToSpreadsheetProcess extends JavaProcess
 {
 	final SpreadsheetExporterService spreadsheetExporterService = SpringContextHolder.instance.getBean(SpreadsheetExporterService.class);
+	private	SpreadsheetFormat spreadsheetFormat;
+
+	@Override
+	protected void prepare()
+	{
+		spreadsheetFormat = getProcessInfo().getSpreadsheetExportOptions().getFormat();
```

Comment:
why do we need this line?
i think you can loose this whole `prepare` method and use `@Param` instead. See e.g. `de.metas.impexp.process.C_DataImport_ImportAttachment#p_AD_AttachmentEntry_ID`

Reply:
modification done